src/providers/FusionAuthProvider.tsx

```diff
@@ -56,35 +71,39 @@ export const FusionAuthProvider: React.FC<Props> = ({ config, children }) => {
             const code = await generatePKCE();
             Cookies.set('codeVerifier', code.code_verifier);
             const queryParams = {
-                client_id: config.clientID,
-                scope: config.scope,
+                client_id: props.clientID,
+                scope: props?.scope ?? DEFAULT_SCOPE,
                 response_type: 'code',
-                redirect_uri: config.redirectUri,
+                redirect_uri: props.redirectUri,
                 code_challenge: code.code_challenge,
                 code_challenge_method: 'S256',
                 state: stateParam,
             };
             const fullUrl = generateUrl(FunctionType.login, queryParams);
             window.location.assign(fullUrl);
         },
-        [config, generateUrl],
+        [generateUrl, props.clientID, props.redirectUri, props?.scope],
     );
 
     const logout = useCallback(async () => {
+        const accessToken = Cookies.get('access_token');
         // Clear cookies
         Cookies.remove('user');
         Cookies.remove('lastState');
         Cookies.remove('codeVerifier');
-        Cookies.remove('refresh_token');
-        Cookies.remove('access_token');
+        // Cookies.remove('refresh_token');
+        // Cookies.remove('access_token');
```

Comment:
Should these be commented out or just deleted?

Reply:
Removed--these cookies are supposed to be HTTP only/secure so are not visible from the client (only the browser sees them). I have some updates on the example app as well that have these changes